.circleci/config.yml

```diff
@@ -100,6 +100,7 @@ workflows:
               # update the value of PPS_BUCKETS above
               - ADMIN
               - AUTH
+              - ENTERPRISE
```

Comment:
Move the enterprise tests out of the miscellaneous bucket, which is getting slow, into a new bucket for the enterprise and license tests.